src/generate-from-template.js

```diff
@@ -43,13 +44,17 @@ const generateFromTemplate = async ({ templateName }) => {
 
   // Gather parameters
   log.goToStep(2, templateBanner);
+  const storedValues = getStoredValues(templatePackageName);
```

Comment:
you should wrap this in `if (generatorOptions.storeResponses) {` too

if v1 of a template chooses to store values, but v2 does not, then you should not load values that were stored by v1.

Reply:
I don't think this is possible since we get the `generatorOptions` right after this line. And we need to fetch stored values before we call `getTemplateOptions` to be able to pass in the stored values.

Reply:
Added this version storage functionality